### PR TITLE
Don't display '0% 0.00/0.000' for tanks without "capacity" and "remai…

### PR DESCRIPTION
**Pull request by DanielMcInnes**
…ning" values

### CODE REVIEW

File: components/TankGaugeGroup.qml [hunk 1]
```diff
@@ -137,6 +137,7 @@ Rectangle {
 		width: Theme.geometry_levelsPage_gaugeDelegate_contentWidth
 		font.pixelSize: Theme.font_size_h1
 		unit: VenusOS.Units_Percentage
+		visible: !isNaN(root.level)
```
Comment (blammit) [inline components/TankGaugeGroup.qml:140]: The level should always be present though, shouldn't it?

**Review by blammit — COMMENTED** [review]
Part of the bug is also that the overall levels value is incorrect when there are multiple tanks without capacity/remaining -- were you able to reproduce this? (Even with mock values?)

File: components/TankGaugeGroup.qml (hunk 1 above)
Comment (DanielMcInnes) [reply components/TankGaugeGroup.qml:140]: I don't think so. If you look at the picture from the slack thread, the percentage text is 0%, while the gauge is showing some other non-zero value
image

**Comment by DanielMcInnes** [comment]
> Part of the bug is also that the overall levels value is incorrect when there are multiple tanks without capacity/remaining -- were you able to reproduce this? (Even with mock values?)

No I wasn't. There is something screwy with the mock tanks - if you hard code the TankGauge value to 0.5:
`
// TankGaugeGroup.qml
value: 0.5 // (root.mergeTanks ? model.device.level : root.level) / 100`

... the gauges jump all over the place. If I run the same code in a wasm build, it behaves as you would expect, every gauge is at 50%.

Martin gave me the vrm portal id for the installation that reported the issue. I want to test against the real installation before I merge this. The site is currently offline.

**Comment by DanielMcInnes** [comment]
I'll leave this PR back in 'draft' until I can test against the real installation (currently offline) and verify that it looks right

**Comment by blammit** [comment]
This PR is obsoleted by https://github.com/victronenergy/gui-v2/pull/854, closing.

**Pull request closed without merging**